modules/installation-osp-default-deployment.adoc

```diff
@@ -11,9 +11,9 @@ endif::[]
 [id="installation-osp-default-deployment_{context}"]
 = Resource guidelines for installing {product-title} on OpenStack
 
-Your quota must meet the following requirements to run the {product-title} installation program in RHOSP:
+Your quota must meet the following requirements to run the {product-title} installation program in {rh-openstack}:
```

Comment:
This should probably be the {rh-openstack-first} attribute since it's the first instance in the module.